src/Handler/Project/Create/CreateProjectHandler.php

```diff
@@ -120,12 +120,11 @@ public function __invoke(
         $fileStorageBucket = $storageManager->bucket($fileStorageBucketName);
         $actualBucketPolicy = $fileStorageBucket->iam()->policy();
 
-        $actualBucketPolicy['bindings'][] = [ // project service account can list and get files
-            'role' => 'roles/storage.objectViewer',
-            'members' => ['serviceAccount:' . $projectServiceAccount->getEmail()],
-        ];
-        $actualBucketPolicy['bindings'][] = [ // project service account can export to bucket
-            'role' => 'roles/storage.objectCreator',
+        // project service account can list and get files
+        // project service account can export to bucket
+        // project service account can delete files, needed also for export
+        $actualBucketPolicy['bindings'][] = [
+            'role' => 'roles/storage.objectAdmin',
```

Comment:
Jo super vedel som ze niekam som potreboval pridat pravo na to mazanie ale nevedel som uz kde

Reply:
Jo divil sem sa, žes zavřel ten task a že mě to nefunguje :)

Reply:
ja som prave vcera premyslal nad itm kde je to potreba. vravim si hmm vsetko mi funguje tak mozno nejaka haluz ze som to niekde nemal :) Tak uz viem kde to malo byt :D

Reply:
Ono je to potřeba ještě na mazání souborů z file storage a mazání snapshotů